### PR TITLE
Onboarding content audit part 1

### PR DESCRIPTION
**Pull request by thoragudf**
Doing an onboarding content audit. This is part 1 where I'm making changes to our docs front page and related content. 

See what's next up here: https://app.asana.com/0/1207301331218898/1207301331218898

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/audit/_meta**](https://docs-815mm7j87.avo-dev.app/audit/_meta) | [**…tracking-plan-into-avo**](https://docs-815mm7j87.avo-dev.app/audit/get-tracking-plan-into-avo) |
|:-:|:-:|
|<a href="https://docs-815mm7j87.avo-dev.app/audit/_meta">Screenshot of /audit/_meta</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/-h7fUUAyezzET03jyfvSuI_Ij_xSU1vss6iCFNbNZxmD4ltnZQlPp2jItDklzVS_o7hGCtG1lQRxB_EzYfldkg.png">(view full size)</a> |<a href="https://docs-815mm7j87.avo-dev.app/audit/get-tracking-plan-into-avo">Screenshot of /audit/get-tracking-plan-into-avo</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/hqFsna8N6VLcGZqyPKE_l8h-O5xi6SYAYBFXYAqcvqRTvYVCjkTsEaDLmcqGNgouxoQiKXjNzAEhn5woj-LGhQ.png">(view full size)</a> |



| [**/**](https://docs-815mm7j87.avo-dev.app/) |
|:-:|
|<a href="https://docs-815mm7j87.avo-dev.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Dev8Y346ERp4mDrP1CxPaRju-C9XKWb-D4eZl5oRMIMHZg4iOZ_-J0woEIGkDzwRUSmJCe2wNoI7rxpv_PRudg.png">(view full size)</a> |







Commit 53f593e92f408338e7060e70cf4fb675856b9210 (https://docs-815mm7j87.avo-dev.app).

File: pages/index.mdx [hunk 1]
```diff
@@ -7,23 +7,30 @@ import PageLink, { TwoCol } from '../components/PageLink';
...
+There are a few ways to get started in Avo, all depending on your immediate needs:
+- [**Find (and fix) your data quality issues**](#find-and-fix-your-data-quality-issues): For teams that are struggling with data quality, install Inspector and get a view of all your data quality issues in one place
+- [**Try the Avo workflow**](#try-the-avo-workflow-for-your-next-analytics-release): For teams that want a solid workflow for their next analytics release, create a branch in Avo and define new events and properties
+- [**Audit your tracking plan**](#audit-your-tracking-plan): For teams that want a single source of truth for their tracking plan and review their event and property definitions, import your existing tracking plan into Avo or manually create one
+
+You can do these steps in any order or in parallel, and skip those you
 don’t think apply to you.
 
-## Get an overview of your current state of tracking with Inspector
+## Find (and fix) your data quality issues
```
Comment (hugihlynsson) [inline pages/index.mdx:18]: Love it!

File: pages/audit/get-tracking-plan-into-avo.mdx
Comment (hugihlynsson) [inline pages/audit/get-tracking-plan-into-avo.mdx:1]: Should we change the wording on this page to say "**existing** Tracking plan" or is that implicit?

**Review by hugihlynsson — APPROVED** [review]
Solid improvements!

File: pages/audit/get-tracking-plan-into-avo.mdx
Comment (thoragudf) [reply pages/audit/get-tracking-plan-into-avo.mdx:1]: I think it's good to be clear. Great suggestion!

**Pull request merged**